engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/DeserializeLibrarySuggestionsJob.scala

```diff
@@ -36,11 +37,13 @@ final class DeserializeLibrarySuggestionsJob(
     cc
       .deserializeSuggestions(libraryName)
       .foreach { cachedSuggestions =>
+        val suggestions =
+          cachedSuggestions.asInstanceOf[java.util.List[Suggestion]]
```

Comment:
```suggestion
      .foreach { case suggestions: java.util.List[Suggestion] =>
```
or sth like that (probably needs to deal with an erasure warning as well)

Reply:
I tried to apply the suggestion, but:
```
enso/engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/DeserializeLibrarySuggestionsJob.scala:40:46: non-variable type argument org.enso.polyglot.Suggestion in type pattern java.util.List[org.enso.polyglot.Suggestion] is unchecked since it is eliminated by erasure
[error]       .foreach { case suggestions: java.util.List[Suggestion] =>
```
I prefer code that compiles over one that doesn't. Returning back `cachedSuggestions.asInstanceOf[java.util.List[Suggestion]]`